### PR TITLE
[INLONG-1015][Audit] Add internal data protocol documentation for Audit

### LINKED ISSUE #1015: [Improve][Audit] Add internal data protocol documentation for Audit
### Description

Add internal data protocol documentation for Audit

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
- Fixes #1015 

### Motivation

Add internal data protocol documentation for Audit

### Modifications
Added a new document to introduce the composition of the audit system data protocol and the final table schema information

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+title: Audit 数据格式定义及使用
+sidebar_position: 2
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md:3]:
```suggestion
sidebar_position: 5
```

**Review by justinwwhuang — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    FAILED = 1;
+    DISASTER = 2;
+  }
+  uint64 request_id = 1;
+  RSP_CODE rsp_code = 2;
+  string message = 3;
+}
+```
+
+##### Audit Header
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md:83]:
```suggestion
#### Audit Header
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md (hunk 1 above)
Comment (doleyzi) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md]: Fixed

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md (hunk 2 above)
Comment (doleyzi) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/audit_msg.md]: Fiexd

**Review by aloyszhang — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Pull request merged**